docs/mkdocs/docs/api/basic_json/operator_eq.md

```diff
@@ -46,7 +56,11 @@ Linear.
 
 !!! note "Comparing special values"
 
-    - NaN values never compare equal to themselves or to other NaN values.
+    - `NaN` values are unordered within the domain of numbers.
+      The following comparisons all yield `#!cpp false`:
```

Comment:
Is it not true that all equality comparisons with a `NaN` value return `false` now?  If that's true, then is there a need to call out specific things which return `false`?

Reply:
No. See https://github.com/nlohmann/json/issues/3409#issuecomment-1100829527 where I asked about this specifically. What you state was initially true, but I realized, that that's nonsensical.

Reply:
Makes sense, thanks.